vignettes/dettl.Rmd

```diff
@@ -111,11 +111,11 @@ The process at a minimum needs code to define 3 functions one for each stage of
 
 ## Creating a new ETL process
 
-If you want to create a new process within an existing project this can be done by opening the project as an RStudio project and using `new_dettl()`. Note that the use of `withr::with_dir(tempdir(), ...)` in the example below is because this is running in a vignette and the code is an example. This is not necessary when using `dettl` in a real environment.
+If you want to create a new process within an existing project this can be done by opening the project as an RStudio project and using `dettl_new()`. Note that the use of `withr::with_dir(tempdir(), ...)` in the example below is because this is running in a vignette and the code is an example. This is not necessary when using `dettl` in a real environment.
```

Comment:
Not strictly related to this PR but the "new process" here is confusing me because I read [instance of a computer program](https://en.wikipedia.org/wiki/Process_(computing))

Reply:
Yes that is a bit confusing. And I think talking about it as a "process" is not really a thing that occurs anywhere other than in the vignette. I've updated that in favour of talking about and import.